openfecli/tests/commands/test_plan_rbfe_network.py

```diff
@@ -149,6 +152,43 @@ def test_plan_rbfe_network_cofactors(eg5_files):
 
         assert result.exit_code == 0
 
+@pytest.fixture
+def cdk8_files():
+    with resources.files("openfe.tests.data.cdk8") as p:
+        pdb_path = str(p.joinpath("cdk8_protein.pdb"))
+        lig_path = str(p.joinpath("cdk8_ligands.sdf"))
+
+        yield pdb_path, lig_path
+
+def test_plan_rbfe_network_charge_changes(cdk8_files):
+    """
+    Make sure the protocol settings are changed and a warning is printed when we plan a network
+    with a net charge change.
+    """
+    runner = CliRunner()
+
+    args = [
+        '-p', cdk8_files[0],
+        '-M', cdk8_files[1],
+    ]
+
+    with runner.isolated_filesystem():
+        with pytest.warns(UserWarning, match="Charge changing transformation between ligands lig_40 and lig_41"):
+            result = runner.invoke(plan_rbfe_network, args)
+
+            print(result.output)
```

Comment:
I don't think we need this print statement here.

Reply:
Good catch, this is what happens when I copy and paste a test!

Reply:
I still see the print statement, just putting it here in case you forgot to remove it, if not, please ignore this comment!